### PR TITLE
Issue #4362: Google-site help pages

### PR DESCRIPTION
**Pull request by felixgondwe**
Updated help links to point to google site help pages.

### CODE REVIEW

**Comment by ragingsquirrel3** [comment]
please remove the dump.rdb file

File: src/sgd/frontend/yeastgenome/static/templates/curator_sequence.jinja2 [hunk 1]
```diff
@@ -38,7 +38,7 @@
 <!-- Overview
 ================================================== -->
     <section id="overview" data-magellan-destination="overview">
-        <a href="/help/sequence" name="top" class="documentation-link">Sequence Help <i class="fa fa-question-circle"></i></a>
+        <atarget="_blank" href="https://sites.google.com/view/yeastgenome-help/sequence-help" name="top" class="documentation-link">Sequence Help <i class="fa fa-question-circle"></i></a>
```
Comment (ragingsquirrel3) [inline src/sgd/frontend/yeastgenome/static/templates/curator_sequence.jinja2]: example of needed space

File: src/sgd/frontend/yeastgenome/static/templates/literature_details.jinja2 [hunk 2]
```diff
@@ -42,7 +42,7 @@
 <!-- Overview
 ================================================== -->
 <section id="overview" data-magellan-destination="overview">
-    <a href="/help/literature-help/literature-guide" name="top" class="documentation-link">Literature Help <i class="fa fa-question-circle"></i></a>
+    <atarget="_blank" href="https://sites.google.com/view/yeastgenome-help/literature-help/literature-page" name="top" class="documentation-link">Literature Help <i class="fa fa-question-circle"></i></a>
```
Comment (felixgondwe) [inline src/sgd/frontend/yeastgenome/static/templates/literature_details.jinja2]: added space

File: src/sgd/tools/blast/__init__.py [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import json
+from urllib2 import Request, urlopen, URLError
```
Comment (felixgondwe) [inline src/sgd/tools/blast/__init__.py]: reverted change, it was a pylint suggestion.

File: .gitignore [hunk 4]
```diff
@@ -26,8 +26,8 @@ client/lib/sgd_visualization/.git
...
 asset_version.json
-.idea/
-settings.json
-launch.json
 .vscode
-
+.vscode/
+.idea/
+*.rdb
+.vscode/settings.json
```
Comment (ragingsquirrel3) [inline .gitignore]: very small thing, but try to always leave a blank line at the end of a file

**Comment by ragingsquirrel3** [comment]
Looks good, thanks!

**Pull request merged**